frontend/src/app/components/ExperimentPage.js

```diff
@@ -34,7 +35,8 @@ export default class ExperimentPage extends React.Component {
       progressButtonWidth: null,
       showDisableDialog: false,
       shouldShowTourDialog: false,
-      showTourDialog: false
+      showTourDialog: false,
+      showPreFeedbackDialog: false
```

Comment:
It's getting kinda hectic, so we can borrow some tech-debt and make #1367 bigger IMO